File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/command/uielement/ButtonUi.java
package com.bigboxcode.designpattern.command.uielement;

public class ButtonUi implements UiCommand {

    private final String name;

    public ButtonUi(String name) {
        this.name = name;
    }

    @Override
    public void print() {
        System.out.println("Printing " + this.name + " Button");
    }

    @Override
    public void remove() {
        System.out.println("Removing " + this.name + "  Button");
    }
}

File: design-patterns/php/command/uielement/ButtonUi.php
<?php
// ButtonUi.php

namespace BigBoxCode\DesignPattern\Command\UiElement;

class ButtonUi implements UiCommand {
    public function __construct(private string $name) {
    }

    public function print(): void {
        echo "Printing " . $this->name . " Button\n";
    }

    public function remove(): void {
        echo "Removing " . $this->name . "  Button\n";
    }
}